openfisca_aotearoa/tests/social_security/sole_parent_support/sole_parent_support__split_care.yaml

```diff
@@ -0,0 +1,303 @@
+- name: Sole Parent Split Care, no one eligible - why is this false and not incalculatable?
+  period: 2019-W05
+  absolute_error_margin: 0
+  input:
+    persons:
+      Koro:
+        age:
+          day:2019-01-28:7: 50
+        social_security__in_a_relationship: false
+        social_security__residential_requirement: true
+        sole_parent_support__parents_living_apart: true
+        social_security__principal_caregiver:
+          month:2019-01:2: true
+      Mike:
+        sole_parent_support__parents_living_apart: true
+        social_security__principal_caregiver:
+          month:2019-01:2: true
+        age:
+          day:2019-01-28:7: 50
+      Tamaiti:
+        age:
+          day:2019-01-28:7: 5
+      Ruru:
+        age:
+          day:2019-01-28:7: 3
+    families:
+      Whanau:
+        principal: Koro
```

Comment:
Here as you're using _social_security__principal_caregiver_ you could instead declare Koro as parent, no?